renderer.js

```diff
@@ -78,12 +77,12 @@ var symbolsListNumber = $('#symbols-list-number>li');
 function hoverChangeSection(tab){
   for (var i = 0; i < tab.length; i++) {
     $(tab).eq(i).attr('data-text', i);
-    $(tab).eq(i).on('mouseover', function(){
+    $(tab).eq(i).on('mouseover', function () {
```

Comment:
space in here looks unnecessary，but I will merged this feature. you should keep in mind it.